u-benchmarks/sys_test.go

```diff
@@ -2,55 +2,791 @@ package main
 
 import (
 	"log"
+	"net"
 	"os"
 	"testing"
+	"time"
 )
 
+const ipport = "127.0.0.1:12345"
+
 func BenchmarkFileOpen(b *testing.B) {
+	f := CreateOrDie("hello_world.txt")
```

Comment:
Maybe we want to make this filename a const as well? It is used atleast 25 times from what I could tell.